dns/nixcon.org.js

```diff
@@ -0,0 +1,43 @@
+D("nixcon.org",
+	REG_NONE,
+	DnsProvider(DSP_GANDI),
+
+	MX("@", 10, "umbriel"),
```

Comment:
The MX record set here translates to `umbriel.nixcon.org` which is non-existing currently. Should it maybe be `umbriel.nixos.org`?

Reply:
Yes, it seems clear.  Thanks!  https://github.com/NixOS/infra/pull/647

Reply:
Deployed.